scripts/legacy_expression_tool/README.md

```diff
@@ -82,24 +82,24 @@ python legacy_expression_tool.py \
-| `--service`            | The fiware service filter to replace the expressions                                          | All subservices              | No        |
-| `--service-path`       | The fiware service path filter to replace the expressions                                     | All subservices              | No        |
-| `--deviceid`           | The device id filter to replace the expressions                                               | All devices                  | No        |
-| `--entitytype`         | The entity type filter to replace the expressions                                             | All entity types             | No        |
-| `--regexservice`       | The fiware service regex filter to replace the expressions                                    | All subservices              | No        |
-| `--regexservicepath`   | The fiware service path regex filter to replace the expressions                               | All subservices              | No        |
-| `--regexdeviceid`      | The device id regex filter to replace the expressions                                         | All devices                  | No        |
-| `--regexentitytype`    | The entity type regex filter to replace the expressions                                       | All entity types             | No        |
+| Argument               | Description                                                                                              | Default value                | Mandatory |
+| ---------------------- | -------------------------------------------------------------------------------------------------------- | ---------------------------- | --------- |
+| `--mongouri`           | The MongoDB URI to connect to                                                                            | `mongodb://localhost:27017/` | No        |
+| `--database`           | The database name to replace the expressions                                                             | NA                           | Yes       |
+| `--collection`         | The collection name to replace the expressions                                                           | NA                           | Yes       |
+| `--translation`        | The translation dictionary file to replace the expressions                                               | `translation.json`           | No        |
+| `--debug`              | Enable debug mode                                                                                        | `False`                      | No        |
+| `--commit`             | Commit the changes to the database                                                                       | `False`                      | No        |
+| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete`, `ignore`, `jexl` or `jexlall` | `ignore`                     | No        |
```

Comment:
```suggestion
| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete` (to delete the field), `ignore` (to leave the field untouched), `jexl` (...) or `jexlall` (...) | `ignore`                     | No        |
```

The `(...)` has to be filled. Not sure the difference between `jexl` and `jexlall` so I cannot complete myself...

Reply:
```suggestion
| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete` (to delete the field), `ignore` (to leave the field untouched), `jexl` (to force to jexl on all groups that match the conditions) or `jexlall` (that, additionally, includes all groups that have a different expression language value than jexl to the query) | `ignore`                     | No        |
```

Reply:
```suggestion
| `--expressionlanguage` | What to do with the expression language field. Possibles values: `delete` (to delete the field), `ignore` (to leave the field untouched), `jexl` (to force to jexl on all groups/devices that match the conditions) or `jexlall` (like `jext` but additionally including all groups/devices that have a different expression language value than jexl to the query) | `ignore`                     | No        |
```